edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.IoTHub/sdkClient/WrappingSdkModuleClient.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Agent.IoTHub.SdkClient
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Client;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Azure.Devices.Shared;
+
+    public class WrappingSdkModuleClient : ISdkModuleClient
```

Comment:
Why the wrapper?

Reply:
@varunpuranik explained that this pass-through simply adds a layer of abstraction that can be used for mocking the SDK in tests.